unity-renderer/Assets/Rendering/UI/Rainbow/UIRadialRainbow.cs

```diff
@@ -0,0 +1,89 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class UIRadialRainbow : MonoBehaviour
+{
+    Material _targetMat;
```

Comment:
In order to follow our code conventions, lets try to avoid the use of `_` in the variables:

```suggestion
    Material targetMat;
```